client/damage/damage.lua

```diff
@@ -204,11 +211,12 @@ local function checkForDamage()
     if isArmorDamaged or isHealthDamaged then
         local damageDone = (Hp - health)
         local weaponHash = applyDamage(cache.ped, damageDone, isArmorDamaged)
-        if weaponHash and not WeaponsThatDamagedPlayer[weaponHash] then
+                -- Add safety check for WEAPONS table and weaponHash ! Its very useful 🙌
```

Comment:
This comment isn't needed.

Reply:
I added this comment that Mods can understand what I did. Mods have power to change my code freely remove the comment

Reply:
I removed those unwanted comments you can accept my pull request now